tools/lint/lint.py

```diff
@@ -475,7 +475,7 @@ def check_parsed(repo_root: Text, path: Text, f: IO[bytes]) -> List[rules.Error]
 
     testdriver_vendor_nodes: List[ElementTree.Element] = []
     if source_file.testdriver_nodes:
-        if test_type != "testharness":
+        if test_type not in {"testharness", "reftest", "print-reftest", "crashtest", "support"}:
```

Comment:
Why we need `support` here? for reftest references? Maybe add a comment for that.

Reply:
Actually, now that I think about it, I'm not sure why `testdriver-child.html` and others are in the manifest in the first place, since similar cases like `clipboard-apis/resources/page.html` aren't in `support` (so presumably they aren't linted).

Probably fine to not check this rule for `support` files for now.